Competition2017/src/competition/subsystems/servo/ServoSubsystem.java

```diff
@@ -0,0 +1,32 @@
+package competition.subsystems.servo;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XServo;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+@Singleton
+public class ServoSubsystem extends BaseSubsystem {
+    public XServo servo;
+    private DoubleProperty minPose;
+    private DoubleProperty maxPose;
+    private double deltaPose;
+    
+    @Inject
+    public ServoSubsystem(WPIFactory factory, XPropertyManager propManager){
+        servo = factory.getServo(5);
```

Comment:
There are actually two servos here: one on the left shooter and one on the right. Can you add a second servo which takes the same goal? It's worth noting that they are mirrored, so if one has a target of, say, `20%` the other should have a goal of `80%`. I am not sure what units the servo actually operates in -- presumably a percentage from 0 to 1.